plugins/modules/slack.py

```diff
@@ -32,51 +32,42 @@
-      - "WebAPI token:
-        Slack WebAPI requires a personal, bot or work application token. These tokens start with V(xoxp-), V(xoxb-)
-        or V(xoxa-), for example V(xoxb-1234-56789abcdefghijklmnop). WebAPI token is required if you intend to receive thread_id.
-        See Slack's documentation (U(https://api.slack.com/docs/token-types)) for more information."
+      - Slack integration token. This authenticates you to the slack service. Make sure to use the correct type of token, depending on what method
+        you use.
+      - 'Webhook token: Prior to Ansible 1.8, a token looked like V(3Ffe373sfhRE6y42Fg3rvf4GlK). In Ansible 1.8 and above, Ansible adapts to the
+        new slack API where tokens look like V(G922VJP24/D921DW937/3Ffe373sfhRE6y42Fg3rvf4GlK). If tokens are in the new format then slack will
+        ignore any value of domain. If the token is in the old format the domain is required. Ansible has no control of when slack will get rid
+        of the old API. When slack does that the old format will stop working. ** Please keep in mind the tokens are not the API tokens but are
+        the webhook tokens. In slack these are found in the webhook URL which are obtained under the apps and integrations. The incoming webhooks
+        can be added in that area. In some cases this may be locked by your Slack admin and you must request access. It is there that the incoming
+        webhooks can be added. The key is on the end of the URL given to you in that section.'
+      - "WebAPI token: Slack WebAPI requires a personal, bot or work application token. These tokens start with V(xoxp-), V(xoxb-) or V(xoxa-),
+        for example V(xoxb-1234-56789abcdefghijklmnop). WebAPI token is required if you intend to receive thread_id. See Slack's documentation
+        (U(https://api.slack.com/docs/token-types)) for more information."
     required: true
   msg:
     type: str
     description:
-      - Message to send. Note that the module does not handle escaping characters.
-        Plain-text angle brackets and ampersands should be converted to HTML entities (e.g. & to &amp;) before sending.
-        See Slack's documentation (U(https://api.slack.com/docs/message-formatting)) for more.
+      - Message to send. Note that the module does not handle escaping characters. Plain-text angle brackets and ampersands should be converted
+        to HTML entities (for example C(&) to C(&amp;)) before sending. See Slack's documentation (U(https://api.slack.com/docs/message-formatting)) for more.
   channel:
     type: str
     description:
       - Channel to send the message to. If absent, the message goes to the channel selected for the O(token).
   thread_id:
     description:
-      - Optional. Timestamp of parent message to thread this message. https://api.slack.com/docs/message-threading
+      - Optional. Timestamp of parent message to thread this message. https://api.slack.com/docs/message-threading.
```

Comment:
```suggestion
      - Optional. Timestamp of parent message to thread this message, see U(https://api.slack.com/docs/message-threading).
```

Reply:
Should we remove the period after the URL at the end of the line as well? You requested this in this in `sensu_handler` (and maybe another ones) in this very PR.

Reply:
Not adding this to the batch this time around.

Reply:
No, the period should stay; in the sensu_handler module it was `sentence: URL` (no period needed) and not `Sentence, see URL.` (period needed). Simply having `Sentence. URL` is strange with or without period ;)